app/lib/ui/about/about_screen.dart

```diff
@@ -35,7 +35,7 @@ class _AboutScreenState extends State<AboutScreen> {
             child: SingleChildScrollView(
       child: Column(
         children: [
-          Image(image: AssetImage('assets/il_about.png')),
+          Image(image: AssetImage('assets/il_about.png'), fit: BoxFit.cover),
```

Comment:
The `width: double.infinity` is needed to fit full width

Reply:
Done